src/coreclr/src/gc/gcinterface.h

```diff
@@ -7,7 +7,7 @@
 
 // The major version of the GC/EE interface. Breaking changes to this interface
 // require bumps in the major version number.
-#define GC_INTERFACE_MAJOR_VERSION 4
+#define GC_INTERFACE_MAJOR_VERSION 5
```

Comment:
I don't think you need to increase this again, it was already increased with your last checkin to this file. we just need to increase it each time we actually release something

Reply:
I see. I thought we literally have to update every time we merge breaking changes :-)

Reply:
Was a bit surprised the number is so low.

Reply:
every time we make breaking changes that are publicly visible.